plugin-server/src/main/ingestion-queues/session-recording/session-recordings-consumer.ts

```diff
@@ -149,14 +150,18 @@ export class SessionRecordingIngester {
 
         this.realtimeManager = new RealtimeManager(this.redisPool, this.config)
 
+        // We create a hash of the cluster to use as a unique identifier for the high water marks
+        // This enables us to swap clusters without having to worry about resetting the high water marks
+        const kafkaClusterIdentifier = crypto.createHash('md5').update(this.config.KAFKA_HOSTS).digest('hex')
```

Comment:
this is awesome...

2 thoughts

1) we could sort `this.config.KAFKA_HOSTS` it's not going to be readable anyway and that would stop a no-op for a human like a;b;c -> a;c;b invalidating the watermarks

2) do we need to hash them? the key would be very long without a hash but in an incident it might be nicer to have a;b;c than 23e233 to see why something isn't working

(that said: I always think we should get better at avoiding incidents than at making them go well so neither is blocking)

Reply:
I guess we could... just feels a bit nasty, and I'm a little worried about weird characters...